src/e2e/java/uk.gov.hmcts.reform.sscs/RetrieveFileFromSftpService.java

```diff
@@ -26,13 +27,13 @@
     private SftpSshService service;
 
     @Test
-    public void givenAnSftpFile_shouldBeRetrievedAndConvertedToAnInputStream() throws Exception {
+    public void shouldBeRetrievedAndConvertedToAnInputStreamGivenAnSftpFile() throws IOException {
```

Comment:
do we care about the exception thrown in the test? Not a big deal, but don't know really why we cannot leave to exception.

Reply:
sonar says to throw specif exception.